doc/lua_api.md

```diff
@@ -7386,19 +7388,17 @@ Server
     * Details/Notes:
       * If `ephemeral`=false and `to_player` is unset the file is added to the media
         sent to clients on startup, this means the media will appear even on
-        old clients if they rejoin the server.
+        old clients (<5.3.0) if they rejoin the server.
       * If `ephemeral`=false the file must not be modified, deleted, moved or
-        renamed after calling this function.
-      * Regardless of any use of `ephemeral`, adding media files with the same
-        name twice is not possible/guaranteed to work. An exception to this is the
-        use of `to_player` to send the same, already existent file to multiple
-        chosen players.
+        renamed after calling this function. This is allowed otherwise.
+      * Adding media files with the same name twice is not possible.
```

Comment:
Missing documentation: What happens if it is attempted anyway?

Reply:
But is already documented to return false on errors. My point is that "do not do $thing" or "it is not possible to do $thing" is clear enough and we don't need to mention that invalid usage causes an error every time